### ISSUE TITLE
Cannot reject Guild Invitation

### ISSUE THREAD

**Issue by Aiseant** [HabitRPG/habitica#1516]
I have been invited to a guild I don't want to join (sorry guys :s) but it seems I cannot reject the invitation ; the button just doesn't work.

I knowI could work around it by accepting the invit then leaving the guild, it's just that the button should work, you know ? :D

**Comment by paglias** [comment]
Good catch! 

@lefnire I gave a look to this code and there are a few problems I think:
- guild invitations are sent as party invitations and if you already one it's going to be overriden
- in controllers/groups.js at line 284 it is searching for a group we already have and it's populating the group members with the user, it shouldn't be done when you accept?
- accepting a guild invitation in angular sent a "group" parameter which shouldn't work, it should be "invitation"

If I'm right then joining guilds would have been impossible and someone would have reported it, so probably I'm wrong. I created a pull request with some changes #1517 but check it carefully before merging!
